docs/reference/warehouse/wh_cloud.asciidoc

```diff
@@ -0,0 +1,46 @@
+== Cloud warehouse
```

Comment:
Do we still want to keep this file?

Reply:
@gtback, I intended to keep this file because these settings were not surfaced anywhere specifically in the ES Reference.

Reply:
@debadair, I created a new [Index management settings](https://elasticsearch_57953.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/index-management-settings.html) page to house some of these orphan settings that didn't fit well into existing categories. The page is somewhat of a miscellaneous index bucket, but I didn't want to bury these settings or shoehorn them into pages where they don't fit.

Another consideration is adding them to the [ILM settings](https://elasticsearch_57953.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ilm-settings.html) page as Miscellaneous index settings_ (like in the [Cluster-level settings page](https://elasticsearch_57953.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/modules-cluster.html#misc-cluster-settings).

Reply:
I think having them on a separate page is the right call--they aren't really related to ILM, they are more akin to the cluster-level settings. (In fact, the one *is* a cluster setting.) Since they affect how you work with indices, though, grouping them under Index management seems reasonable.